plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,20 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewGauge(
+		prometheus.GaugeOpts{
```

Comment:
It depends on what you need. Histogram gives you XX percentile over some sliding window, which is our SLI, and thus something that we want to optimize for.